tools/doc/hawqregister_help

```diff
@@ -0,0 +1,97 @@
+COMMAND NAME: hawq register
+
+Register parquet files generated by other system into the corrsponding table in hawq
+
+*****************************************************
+SYNOPSIS
+*****************************************************
+
+hawq extract [-h hostname] [-p port] [-U username] <databasename> <tablename> <hdfspath>
+
+hawq extract help 
+hawq extract -? 
+
+hawq extract --version
+
+*****************************************************
+DESCRIPTION
+*****************************************************
+
+Hawq register si a utility to register file(s) in hdfs into
+the table in hawq. It will move the file in the path(if path
+refers to a file) or files under the path(if path refers to a
+directory) into a new place corresponding to the table <tablename>,
+and then update the meatadata in the table.
+
+To use hawq register, HAWQ must have been started.
+
+Currently hawq register supports parquet file(s) and table only.
+User have to make sure that the structure of the parquet file(s)
+and the table are the same.
+
```

Comment:
Currently "hawq register" supports parquet tables only. Users have to make sure that the meta data of the parquet file(s) and the table are consistent.